### DISCUSSION TITLE
Use special user to create objectbucket not use gen user.

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#11188] [Q&A]
We want to more bucket with same account.

**Comment by travisn** [comment]
To be clear, you want the following?
- Create an object user "A"
- Create multiple OBCs that have the same user A instead of generating new users for each of the buckets

It's a common request, but I don't believe it's possible today. @BlaineEXE ?

**Reply by BlaineEXE** [reply]
That's right. It is a request we've had , but it's not possible today.

**Reply by BlaineEXE** [reply]
What is your use-case for the request?

**Reply by zhucan** [reply]
Facilitate upper-layer application development and docking， no need to change ak and sk for different bucket.

**Reply by BlaineEXE** [reply]
Would you care to be more specific about what kind of application? I can provide this feedback to the development team for the next object bucket interface, but I need clear details about what kind of application requires the feature, who the user is in the scenario, and why having different credentials per bucket is insufficient. It would also be good to understand if adding IAM policy support would effectively allow what you desire.

**Reply by zhucan** [reply]
The same user has different bucket,  one bucket is used to product and other bucket is used to test, the applictaion is harbor

**Reply by BlaineEXE** [reply]
It is helpful to understand the specifics that this is for harbor.

**Reply by BlaineEXE** [reply]
Could you provide more specifics about how harbor is configured in the scenario? I'm not familiar, and having as many specifics as possible will help me.

**Reply by zhucan** [reply]
if we can not create the bucket with special user, the "object-user" is used for? @BlaineEXE

**Comment by zhucan** [comment]
we want to use  special ak and sk to create object user and set bucket's owner with this user.
